packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -0,0 +1,844 @@
+    '
+  />
+  {{! prettier-ignore-end }}
+  <p class="dummy-paragraph">Renders to:</p>
+  <Hds::Alert @type="inline" @color="success" @icon="bulb" @title="Title here" @description="Description here" />
+  <p class="dummy-paragraph">
+    If instead you want to completely hide the icon you have to pass a
+    <code class="dummy-code">false</code>
+    value to the
+    <code class="dummy-code">icon</code>
+    argument.
+  </p>
+  {{! prettier-ignore-start }}
+  {{! template-lint-disable no-unbalanced-curlies }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::Alert @type="inline" @color="success" @icon=\{{false}} @title="Title here" @description="Description here" />
+    '
+  />
+  {{! template-lint-enable no-unbalanced-curlies }}
+  {{! prettier-ignore-end }}
+  <p class="dummy-paragraph">Renders to:</p>
+  <Hds::Alert @type="inline" @color="success" @icon={{false}} @title="Title here" @description="Description here" />
+
+  <h4 class="dummy-h4">Close</h4>
+  <p class="dummy-paragraph">
+    In some cases the alert needs to be dismissable. In this case you have to pass a callback function to the
+    <code class="dummy-code">onClose</code>
+    argument. This will also automatically add a "close/dismiss" button to the alert, that when clicked will execute the
+    callback function.
+  </p>
+  <p class="dummy-paragraph">
+    🚨
+    <em><strong>Important</strong>: the actual implementation of what happens to the alert when the callback function is
+      invoked is left to the developer (this will likely depent on the type of alert, on the context of where it's used,
+      on the specific use case, etc.).</em>
+  </p>
+  {{! prettier-ignore-start }}
+  {{! template-lint-disable no-unbalanced-curlies }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::Alert
+        @type="inline"
+        @color="warning"
+        @title="Title here"
+        @description="Description here"
+        @onClose=\{{ your function here }}
+      />
+    '
+  />
+  {{! template-lint-enable no-unbalanced-curlies }}
+  {{! prettier-ignore-end }}
+  <p class="dummy-paragraph">Renders to:</p>
+  <Hds::Alert
+    @type="inline"
+    @color="warning"
+    @title="Title here"
+    @description="Description here"
+    @onClose={{this.noop}}
+  />
+
+  <h4 class="dummy-h4">Actions</h4>
+  <p class="dummy-paragraph">Actions can optionally be passed into the
+    <code class="dummy-code">&lt;:actions&gt;</code>
+    named block using one of the suggested
+    <code class="dummy-code">Button</code>,
+    <code class="dummy-code">Link::Standalone</code>
+    or
+    <code class="dummy-code">LinkTo::Standalone</code>
+    yielded components.</p>
+  <p class="dummy-paragraph"><code class="dummy-code">Button</code>
+    is a yielded
+    <code class="dummy-code">HDS::Button</code>
+    component, so it exposes exactly
+    <LinkTo @route="components.button">the same API</LinkTo>
```

Comment:
[issue] Links with the same name must serve the same purpose. If they go to different URLs they need to have different names. In this case, I would change the text to "the button component's API" (or even exclude this section entirely, as I am not sure it provides any additional clarity or value). 

There are a few instances of this issue in this page. 

<img width="669" alt="image" src="https://user-images.githubusercontent.com/4587451/166324400-af4ff2cb-74df-4568-9605-09f939a8d806.png">

Reply:
fixed